bigquery-storage/api/BigQueryStorage.Samples/BigQueryStorage.Samples.csproj

```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.1</TargetFramework>
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Google.Cloud.BigQuery.Storage.V1" Version="2.5.0" />
+  </ItemGroup>
+  <ItemGroup>
```

Comment:
I suspect this ItemGroup and the one below it aren't necessary.

Reply:
Yes. Corrected it.